app/Http/Resources/CharacterResource.php

```diff
@@ -17,7 +17,8 @@ public function toArray($request)
         /** @var Character $model */
         $model = $this->resource;
 
-        $raceIDs = $model->characterRaces->pluck('race.id');
+        $raceIDs = $model->characterRaces->where('is_private', false)->pluck('race.id');
```

Comment:
Races should still include all races to avoid people having to change their external tools